README.md

```diff
@@ -267,18 +267,7 @@ To maintain a safe and inclusive space for everyone to learn and grow, contribut
 <!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->
 
 
-<h2 align="center">Feedback</h2>
 
-If you have any feedback or suggestions please reach out to Project - Maintainer.  
-* [Lalit Kumar Singh](https://github.com/Lalit3716) 
-
-
-Or you can create a  <a href="https://github.com/Vita-App/Vita/issues">issue</a> and mention there , which new features or extension can make this Project more good.
-
-<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->
-
-<br>
-  
```

Comment:
Don't remove this section just make some changes I told you to make in previous review